src/randoop/operation/FieldSetter.java

```diff
@@ -99,17 +95,18 @@ public ExecutionOutcome execute(Object[] statementInput, PrintStream out) {
   }
 
   /**
-   * appendCode generates code for setting a field.
-   * Should look like
-   *   field = <value/variable>;
+   * Generates code for setting a field.
+   * Should look like<br>
+   *   <code>field = value;</code><br>
```

Comment:
If you want code on its own line, `<pre>...</pre>` is more idiomatic than `<br><code>...</code><br>`.